### PR TITLE
Add some docs and comments

### PR DESCRIPTION
**Pull request by aliok**

### CODE REVIEW

File: README.md
Comment (aliok) [inline README.md:81]: Happy to incorporate some docs if you can describe what this is doing

File: README.md
Comment (aliok) [inline README.md:83]: Happy to incorporate some docs if you can describe what this is doing @pierDipi

File: bin/common.sh
Comment (aliok) [inline bin/common.sh:54]: Get rid of stupid warning (kind of irrelevant to the PR)

File: README.md
Comment (pierDipi) [inline README.md:71]: This is not true, both are pointing to the latest already of their respective stream.

File: README.md
Comment (pierDipi) [reply README.md:83]: Let's chat on Slack about this.

File: README.md
Comment (pierDipi) [reply README.md:81]: It's unused because #44 but it can be removed, contributions are welcome!

File: README.md
Comment (pierDipi) [inline README.md:79]: what do you mean?

File: README.md
Comment (pierDipi) [inline README.md:77]: > This is done to make the tests more realistic

This is done to be able to pass the tests, not for the sake of being realistic.

> (NOTE: need to elaborate on the arbitrary numbers and have a scientific basis for them)

can you expand on this?

File: README.md
Comment (pierDipi) [inline README.md:54]: Let's put this into its own section with a separate title.

I usually use the Hyperfoil UI directly by just navigating to the Hyperfoil URL (which is equivalent) to avoid installing the CLI why do you recommend this approach?

File: README.md
Comment (pierDipi) [inline README.md:35]: `oc get hyperfoil` gives you the actual URL without going to the underlying implementation that might change.

File: README.md
Comment (pierDipi) [inline README.md:4]: Worker count can be changed, 15 is the default that is documented in ./bin/run.sh` so I wouldn't duplicate this info here as well.


```suggestion
- Access to an OCP cluster
```

File: README.md
Comment (pierDipi) [inline README.md:7]: It's not really needed since the UI does the same.
```suggestion
```

File: bin/common.sh
Comment (pierDipi) [inline bin/common.sh:203]: Since we added oc 4.10+ as a prerequisite, can we remove this comment here since it's irrelevant for people viewing the code? it might also easily become outdated.

File: README.md
Comment (aliok) [reply README.md:71]: Ok, got it. That's done by subscriptions. Thanks!

File: README.md
Comment (aliok) [reply README.md:54]: That's better. Let me try that (tomorrow)

File: README.md
Comment (aliok) [reply README.md:5]: I haven't tested 4.10. If you know that's good, sure let's change this.

File: bin/common.sh
Comment (pierDipi) [reply bin/common.sh:54]: I remember doing this patch locally but then I've never sent the PR, thanks!

File: README.md
Comment (aliok) [reply README.md]: Removed this TODO. Let's delete `vertx-receiver` in a separate PR.

**Comment by aliok** [comment]
@pierDipi another look?

**Review by pierDipi — APPROVED** [review]
/lgtm

for nit,
/hold

**Comment by pierDipi** [comment]
Thanks for doing this Ali!

**Comment by aliok** [comment]
ok, committed the suggestion, need a new lgtm now.

/unhold

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

**Comment by pierDipi** [comment]
Merging manually, no need to run e2e tests.

**Pull request merged**
